File: Practice Library/Any Language/Sorting and Searching/Insertion Sort/AJs/ins_sort.py
def ins_sort(arr: list):
    i = 1
    while i<len(arr):
        j=i-1
        currentVal = arr[i]
        while (j>=0) and (arr[j]>currentVal):
            arr[j+1]=arr[j]
            j-=1
        arr[j+1]=currentVal
        i+=1
    return arr

print(ins_sort([1,3,4,2,4,5,6,4,1,33])) 

File: Practice Library/Any Language/Sorting and Searching/Insertion Sort/AJs/ins_sort.cs
public int[] sort(int[] list) {
    for (int i = 1; i < list.Count(); i++)
    {
        var currentval = list[i];
        var j=i-1;
        while(j>=0 && list[j]>currentval) {
            list[j+1]=list[j];
            j--;
        }
        list[j+1]=currentval;
    }

    foreach (var item in list)
    {   
        Console.WriteLine(item);
    }
    return list;
}

sort(new [] {1,3,4,2,4,5,6,4,1,33});